### PR TITLE
repaired local config

### PR DESCRIPTION
**Pull request by RobertLaverick**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -176,7 +176,7 @@ external-url {
 accessibility-statement.service-path = "http://localhost:12346/accessibility-statement/get-a-customs-subscription-and-eori-number"
 
 services-config {
-    list = "atar, gagmr, cds, eori-only, ctc, gbss, route1, ars"
+    list = "atar, gagmr, cds, eori-only, ctc, gbss, route1, ars, ivd, ndrc"
```
Comment (JakeGoodwin) [inline conf/application.conf:179]: esc is missing from this list

Comment (burns238) [reply conf/application.conf:179]: Adding a comment so that I can set this to request changes

**Review by burns238 — CHANGES_REQUESTED** [review]

File: conf/application.conf [hunk 2]
```diff
@@ -218,26 +233,38 @@ services-config {
...
-    ars {
-        enrolment = "HMRC-ATAR-ORG"
-        shortName = "ARS"
-        friendlyName = "advance_rulings"
-        friendlyNameWelsh = "ddyfarniadau_uwch"
+    ndrc {
+        enrolment = "HMRC-CTS-ORG",
+        shortName = "CTS",
+        callBack = "http://localhost:8450/apply-for-repayment-of-import-duty-and-import-vat"
+        friendlyName = "Customs_Trader_Services",
```
Comment (burns238) [inline conf/application.conf:256]: some comma separated in here too

Comment (RobertLaverick) [reply conf/application.conf]: I've removed them too

**Comment by RobertLaverick** [comment]
rebuild

**Comment by RobertLaverick** [comment]
test this please

**Review by burns238 — APPROVED** [review]

**Pull request merged**